XUnitTests/UnitTestGCD.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using GrandCommonDivisor.Core;
+using Xunit;
+
+namespace MyFirstUnitTests
+{
+    public class UnitTestGCD
+    {
+        [Theory]
+        [InlineData(1, 2, 1)]
+        [InlineData(-4, -6, 2)]
+        [InlineData(-2, 2, 2)]
+        public void CalculateGCDGoodData_Passing(int value1, int value2, int expected)
+        {
+            Assert.Equal(expected, GCDUtils.CalculateGCD(value1, value2));
+        }
+
+        [Theory]
+        [InlineData(0, 0, 0)]
+        public void CalculateGCDZerosDataFed_Passing(int value1, int value2, int expected)
```

Comment:
I think you can merge merge this test with the previous one, since they both have valid inputs and valid results

Reply:
done